R-package/R/lightgbm.R

```diff
@@ -92,6 +92,11 @@ NULL
 #' @inheritParams lgb_shared_params
 #' @param label Vector of labels, used if \code{data} is not an \code{\link{lgb.Dataset}}
 #' @param weight vector of response values. If not NULL, will set to dataset
+#' @param init_score initial score is the base prediction lightgbm will boost from
```

Comment:
Can you please re-arrange these so the order of the parameters in the roxygen comments matches the function signature? Specifically, `objective` should come before `init_score`, and both should be after `save_name`.

Reply:
Those get rearranged automatically by roxygen. Rearranging them there doesn't guarantee that they'll appear in the same order in the final Rd.

Reply:
I'm glad `{roxygen2}` does that, but even just for the benefit of contributors modifying this code, I'd prefer to keep the documentation in the same order as the function signature.

Reply:
Moved.

Reply:
thank you for moving these, but they are still out of order. `objective` comes before `init_score` in the function signature, but the docs for `init_score` come before `objective. Please make the orders consistent.